dc.prod.yml

```diff
@@ -0,0 +1,61 @@
+version: '3'
+services:
+    reader:
+        build:
+            # TODO: has to be changed before merge
+            # context: "https://github.com/OpenSlides/openslides-datastore-service.git"
+            context: "https://github.com/jsangmeister/openslides-datastore-service.git#fuse-dockerfiles"
+            args:
+                REPOSITORY_URL: "https://github.com/jsangmeister/openslides-datastore-service.git"
+                # GIT_CHECKOUT: "${OPENSLIDES_DATASTORE_SERVICE_COMMIT_HASH}"
```

Comment:
I suggest to use "master" as the default

Reply:
The default is set in the Dockerfile

Reply:
Update: To get rid of the warning displayed if the variable is empty, the defaults are now all set in the `.env` file